src/NuGet.Core/NuGet.Packaging/NupkgMetadata/NupkgMetadataFileFormat.cs

```diff
@@ -53,11 +53,11 @@ public static NupkgMetadataFile Read(TextReader reader, ILogger log, string path
             }
             catch (Exception ex)
             {
-                log.LogWarning(string.Format(CultureInfo.CurrentCulture,
+                var message = string.Format(CultureInfo.CurrentCulture,
```

Comment:
You mentioned we don't want to double log, can you please show an example of how it looked like vs how it looks like now? 

The other thing I'm curious about is how it looks with your other change about failing restore gracefully. 
Should this change be retested with that one?

Reply:
👀

Reply:
Good question. This was more of a conclusion I reached based on the code. We have `log.logWarning` statement followed by an `throw` statement. However, after manually testing the scenario, I did not see any double logs in the console.